docs/content/concepts/layer/control/catalog/_index.md

```diff
@@ -85,157 +88,271 @@ See [examples](#content-examples) of OSCAL catalogs.
 
-This control has a number of high-level data elements, including a security control identifier ("AC-1"), a title ("ACCESS CONTROL POLICY AND PROCEDURES"), the control itself, supplemental guidance, control enhancements, and references. In contrast, the next control is from ISO 27002 on access control policy. It is also detailed in a different way, with an identifier ("9.1.1"), a title ("Access control policy"), control text, lengthy implementation guidance, and other information (additional advice on access control policy).
+Since OSCAL is designed purposefully to represent such data sets (in a specific semantic domain, namely the [NIST Risk Management Framework](https://csrc.nist.gov/publications/detail/sp/800-37/rev-2/final) or RMF), the casting of this information into OSCAL encoding can be straightforward, economical and clean.
 
-This control text is expressed in OSCAL as follows:
+#### Control, ID and title
 
-{{<highlight xml "linenos=table,nowrap=false" >}}
+This control has a number of high-level data elements, including a security control identifier ("AC-1"), a title ("ACCESS CONTROL POLICY AND PROCEDURES"), the control (text) itself, supplemental guidance, control enhancements (although not in this example), and references.
+
+See the source code as currently available in [raw XML from Github](https://raw.githubusercontent.com/usnistgov/oscal-content/main/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_catalog.xml). The control is identified in code with tagging that looks *mainly* as follows, with slight simplifactions. Control AC-1 starts:
+
+{{<highlight xml "nowrap=false" >}}
 <control class="SP800-53" id="ac-1">
-  <title>Access Control Policy and Procedures</title>
-  <param id="ac-1_prm_1">
+    <title>Policy and Procedures</title>
+{{< / highlight >}}
+
+- The control is assigned a class, "SP800-53". This serves as an indicator to a downstream processor of the control's origin (with respect to the catalog or catalog type), with implications for related expectations, including expectations in detail regarding how the control and its contents may be structured.
+- The control ID is "ac-1". All `id` values are unique within the document and serve for addressing and linking. The lexical form of the `id` is designed to be consistent with its formal (conventional) code or label, which is *also* encoded within the control (in this case, "AC-1"). While such regularities are normal and to be expected, this one is a feature of this data set, not a requirement of OSCAL generally.
```

Comment:
Minor, non-blocking: are we still sticking with the terminology document instance? Also be more direct it is not syntactically required but still recommended and preferred?

```suggestion
- The control ID is "ac-1". All `id` values are unique within the document instance and serve for addressing and linking. The lexical form of the `id` is designed to be consistent with its formal (conventional) code or label, which is *also* encoded within the control (in this case, "AC-1"). OSCAL does not require it, but many data set owners benefit from the recommended approach, where lexical and formal control identifiers are consistent and regular in this way.
```